### ISSUE TITLE
[BUG] TypeError: Cannot read properties of undefined (reading 'equalizerNodes')

### ISSUE THREAD

**Issue by ale5000-git** [Andrews54757/FastStream#228]
**Describe the bug**
Hi,
with the latest version I get an error that wasn't present before:
```
Please send this error to the developer at https://github.com/Andrews54757/FastStream/issues
TypeError: Cannot read properties of undefined (reading 'equalizerNodes')
TypeError: Cannot read properties of undefined (reading 'equalizerNodes')
at chrome-extension://kkeakohpadmbldjaiggikmnldlfkdfog/player/ui/audio/AudioChannelMixer.mjs:425:43
at Function.from (<anonymous>)
at AudioChannelMixer.setupNodes (chrome-extension://kkeakohpadmbldjaiggikmnldlfkdfog/player/ui/audio/AudioChannelMixer.mjs:411:31)
at AudioConfigManager.setupNodes (chrome-extension://kkeakohpadmbldjaiggikmnldlfkdfog/player/ui/audio/AudioConfigManager.mjs:359:28)
at FastStreamClient.setSource (chrome-extension://kkeakohpadmbldjaiggikmnldlfkdfog/player/FastStreamClient.mjs:527:33)
at async FastStreamClient.addSource (chrome-extension://kkeakohpadmbldjaiggikmnldlfkdfog/player/FastStreamClient.mjs:449:7)

```
**Version**
1.3.24

**Platform**
Chrome

**Steps to Reproduce**
Url: `ok.ru/video/7100081900128`

**Comment by Andrews54757** [comment]
Your audio profiles are likely corrupted. Can you send me your audio profiles?

**Comment by Yentis** [comment]
I have the same issue, haven't touched anything in audio profiles.
[Profile 1.fsprofile.json](https://github.com/user-attachments/files/16825145/Profile.1.fsprofile.json)

Deleting the default profile resolves the issue.

**Comment by Andrews54757** [comment]
It should be fixed now. It was having trouble migrating legacy profiles

**Comment by ale5000-git** [comment]
@Andrews54757
I never ever changed the audio profile: [Profile 1.fsprofile.json](https://github.com/user-attachments/files/16825166/Profile.1.fsprofile.json)

**Comment by Andrews54757** [comment]
yeah, it's another case of where I fixed one bug and created another. It should be fixed now. You can also fix it by deleting your audio profiles.

**Comment by Andrews54757** [comment]
Fixed in release 1.3.25

**Comment by ale5000-git** [comment]
I confirm that it work perfectly now.
